Reports/Report_doc.Rmd

```diff
@@ -289,10 +289,10 @@ HTML(display_comments(comment_cum))
 
 \newpage
```

Comment:
Without testing metrics we could remove this `\newpage`. Right now, it is producing an extra page for the word doc version of the report:

![image](https://user-images.githubusercontent.com/17878953/105997276-6cf16200-6079-11eb-98a4-625efea1b2ea.png)

Reply:
I will comment it out now